controller/Controller.class.php

```diff
@@ -75,7 +75,10 @@ public static function execute($uri)
         // THIS ROUTE IS DEPRECATED. IT WILL BE REMOVED SOON.
         return static::redirect('https://github.com/lbryio/lbry/releases/download/v0.2.5/lbry_0.2.5_amd64.deb', 307);
```

Comment:
I know this isn't related to the PR, but the deprecation notice on this caught my eye.